### DISCUSSION TITLE
【Bug?】"LEFT OUTER JOIN" does not work correctly when ON condition is 1!=1

### DISCUSSION THREAD

**Discussion by hiroyuki0415** [MarkMpn/Sql4Cds#321] [Q&A]
I have data like below

```sql
SELECT r.crf3c_rank0_3id
FROM   crf3c_rank0_3 AS r;

```

<HTML><head></head>
<BODY>
<TABLE><THEAD><TH>&nbsp;</TH><TH>crf3c_rank0_3id</TH></THEAD><TR><TD ALIGN="center">&nbsp;</TD><TD>2f0384fb-5b14-ee11-9cbe-002248627e12</TD></TR><TR><TD ALIGN="center">&nbsp;</TD><TD>63aecf0d-5c14-ee11-9cbe-002248627e12</TD></TR><TR><TD ALIGN="center">&nbsp;</TD><TD>13a93ef8-5b14-ee11-9cbe-002248627ef1</TD></TR><TR><TD ALIGN="center">&nbsp;</TD><TD>b51e0406-5c14-ee11-9cbe-002248627ef1</TD></TR><TR><TD ALIGN="center">&nbsp;</TD><TD>08a15f0d-5c14-ee11-9cbe-002248627ef1</TD></TR></TABLE>

</BODY>
</HTML>

The result of the SQL below is empty: 
```sql
SELECT r.crf3c_rank0_3id,
       b.businessunitid
FROM   crf3c_rank0_3 AS r
       LEFT OUTER JOIN
       businessunit AS b
       ON 1 != 1;
```

The processing seems to be inner join BUT left outer join.

Thanks in advance for your support.

**Comment by MarkMpn** [comment]
Thanks, yes this looks like a bug with the Nested Loop operator. I've moved this to issue #322 now.
